src/content/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Envoy integration
+tags:
+    - New Relic integrations
+    - Envoy integration
+metaDescription: Use New Relic infrastructure agent to get a dashboard with metrics from your Envoy.
+freshnessValidatedDate: 03/21/2024
+---
+import envoyDashboard from 'images/infrastructure_screenshot_full-envoy-dashboard.webp'
+
+Gain deep insights into Envoy performance with seamless data integration into New Relic. Monitor key metrics to ensure the optimal performance of your Envoy clusters. Create alerts to stay ahead of spikes, build custom dashboards for tailored views, and proactively optimize your database monitoring.
```

Comment:
@sjyothi54 could you help me rewrite this intro? I lack the knowledge about envoy to talk about specifics

Reply:
@bradleycamacho Rephrased the description.